bootstrap/044_brokers.sql

```diff
@@ -1,17 +0,0 @@
-
```

Comment:
pls create a follow-up jira to cleanup the code in the saas too ? maybe, after a week or so

Reply:
Opened https://sundeck.atlassian.net/browse/SD-6841 to track this